preconditions/build.gradle

```diff
@@ -2,7 +2,6 @@ apply from: "${rootDir}/gradle/publish-jar.gradle"
 
 dependencies {
     api project(':safe-logging')
-    api 'com.google.errorprone:error_prone_annotations:2.1.3'
```

Comment:
2.1.3 is older than the version in versions.props, I removed the version in moving to the safe-logging module.